ranking/protocol.go

```diff
@@ -0,0 +1,89 @@
+package ranking
+
+import (
+	"strings"
+
+	"github.com/PretendoNetwork/nex-go"
+	_ "github.com/PretendoNetwork/nex-protocols-go"
+	ranking "github.com/PretendoNetwork/nex-protocols-go/ranking"
+	ranking_mario_kart_8 "github.com/PretendoNetwork/nex-protocols-go/ranking/mario-kart-8"
+	ranking_types "github.com/PretendoNetwork/nex-protocols-go/ranking/types"
+	"github.com/PretendoNetwork/plogger-go"
+)
+
+var commonRankingProtocol *CommonRankingProtocol
+var logger = plogger.NewLogger()
+
+type CommonRankingProtocol struct {
+	server             *nex.Server
+	DefaultProtocol    *ranking.Protocol
+	MarioKart8Protocol *ranking_mario_kart_8.Protocol
+
+	getCommonDataHandler                                     func(unique_id uint64) (error, []byte)
```

Comment:
Reorder these so that the error is the last thing that is returned

```suggestion
	getCommonDataHandler                                     func(unique_id uint64) ([]byte, error)
```